src/Space.cpp

```diff
@@ -541,6 +541,7 @@ Body *Space::FindNearestTo(const Body *b, ObjectType t) const
 
 Body *Space::FindBodyForPath(const SystemPath *path) const
 {
+	if (!m_game->IsNormalSpace()) return nullptr;
 	if (!path->IsSameSystem(m_starSystem->GetPath()))
```

Comment:
Would suggest making this part of the same if-statement for visual clarity, otherwise looks good!

```suggestion
	if (!m_game->IsNormalSpace() || !path->IsSameSystem(m_starSystem->GetPath()))
```

Reply:
Fixed, force-pushed, because the changes are quite simple. Also renamed 'linear orbit' to 'radial orbit' so that others also understand what I mean.